apps/workflow-cf.yml.j2

```diff
@@ -72,6 +72,10 @@ Resources:
             ValueFrom: !Sub "${SecretArn}:username::"
           - Name: EARTHDATA_PASSWORD
             ValueFrom: !Sub "${SecretArn}:password::"
```

Comment:
Instead of *always* providing these, I'm tempted to move them into the `job_spec` task definitions too.

Reply:
I think I'm leaning towards adding these to the `job_spec` task definitions. That way it's very explicit what secrets each plugin is using.